apps-rendering/src/components/Pullquote/index.tsx

```diff
@@ -16,6 +16,7 @@ const styles = (format: ArticleFormat): SerializedStyles => {
 		margin: 0;
 		${headline.xsmall({ fontWeight: 'light' })};
 		${darkModeCss`color: ${text.pullquoteDark(format)};`}
+		margin-bottom: ${remSpace[5]};
```

Comment:
You could combine the margin/margin-bottom css properties. I think the `margin-bottom` is more specific so will win but its not super clear on first glance (I had to google it)

Reply:
Good point. I've just put this in the shorthand in a new commit.